service/src/test/java/greencity/mapping/CategoryDtoResponseMapperTest.java

```diff
@@ -0,0 +1,63 @@
+package greencity.mapping;
+
+import greencity.dto.category.CategoryDtoResponse;
+import org.junit.jupiter.api.Test;
+import greencity.entity.Category;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.modelmapper.ModelMapper;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.*;
+@ExtendWith(SpringExtension.class)
+class CategoryDtoResponseMapperTest {
+    private final ModelMapper modelMapper = new ModelMapper();
+    private final CategoryDtoResponseMapper categoryDtoResponseMapper = new CategoryDtoResponseMapper();
+    @Test
+    void convert_CategoryDtoResponseMapperTest_ShouldMapCorrectly() {
+        Category category = new Category(55L,"Name", "Ім'я", null, null);
```

Comment:
Add a line between the class declaration and fields.
Add a line between a field and a method.